From 21ffb1bb1a55874461c6f9573c0143a86d4ef1be Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 18 Apr 2021 23:28:10 +0300
Subject: [PATCH] [SW-0] boot_desc: include stdint.h for hl_fw_if.h

The hl_fw_if.h is now requiring the stdint.h for portable integer types
like uintxx_t.

Change-Id: Iab29c889e8d2011b53a4fcf8719b5c739a0dadba
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/common/boot_desc.c b/common/boot_desc.c
index a688e82..8cbc974 100644
--- a/common/boot_desc.c
+++ b/common/boot_desc.c
@@ -7,8 +7,9 @@
 #include <stdint.h>
 #include <string.h>
 #include <stdbool.h>
+#include <stdint.h>
 #include <preboot.h>
-#include <hl_fw_if.h>
+#include <hl_fw_if.h>		/* requires stdint.h */
 #include <hl_boot_desc.h>
 #include <common.h>
 
